package.json

```diff
@@ -59,6 +60,12 @@
         "./packages/puppeteer-core:build:docs"
       ]
     },
+    "doctest": {
+      "command": "npx @puppeteer/doctest 'packages/puppeteer-core/lib/esm/**/*.js'",
+      "dependencies": [
+        "./packages/puppeteer-core:build"
```

Comment:
Don't you want to make sure the package is build?

Reply:
Two things to note:
1) `--ignore-scripts` - sometimes I use it.
2) If i change the package itself and run this command it will use the older version, and I think the consensus on the topic is we want it to rebuild in necessary like other test or tools.

Reply:
The problem with `npx` is that it only works if the binary is symlinked in `node_modules/.bin`. This implies if the package was not built, then `npm` has nothing to symlink. We could just do `tools/doctest/bin/doctest.js`, but this creates a dependency on the file name which is what I am preventing.

Open to suggestions.